modules/ROOT/nav.adoc

```diff
@@ -1,40 +1,40 @@
 .xref:index.adoc[Anypoint Monitoring]
 * xref:index.adoc[Overview]
 * xref:monitoring-release-notes.adoc[Release Notes]
-* xref:am-permissions.adoc[Manage Permissions for Anypoint Monitoring]
 * xref:quick-start.adoc[Get Started]
+ ** xref:am-permissions.adoc[Manage Permissions for Anypoint Monitoring]
  ** xref:monitoring-settings-page.adoc[Anypoint Monitoring Settings]
  ** xref:configure-monitoring-cloudhub.adoc[Configure Anypoint Monitoring for Mule Apps (CloudHub)]
  ** xref:monitor-applications-on-rtf.adoc[Monitor Applications on Runtime Fabric]
  ** xref:am-installing.adoc[Monitor Hybrid Runtimes]
  ** xref:performance-and-impact.adoc[Anypoint Monitoring Performance, Impact, and Limits]
 * xref:anypoint-insights.adoc[Anypoint Insights]
-* xref:dashboards.adoc[About Dashboards]
-* xref:dashboards-using.adoc[Use Dashboards]
- ** xref:app-dashboards.adoc[Built-in Application Dashboards]
- ** xref:api-analytics-dashboard.adoc[Built-in API Analytics Dashboard]
- ** xref:create-custom-dashboard.adoc[Create a Custom Dashboard]
- ** xref:export-app-data-to-csv.adoc[Export App Data to a CSV File]
- ** xref:save-builtin-dashboard-as-custom.adoc[Save a Built-in Dashboard as a Custom Dashboard]
-* xref:dashboard-custom-config.adoc[Configure Custom Dashboards and Charts]
- ** xref:dashboard-custom-config-graph.adoc[Configure Graphs]
- ** xref:dashboard-custom-config-singlestat.adoc[Configure Singlestats]
- ** xref:dashboard-custom-config-table.adoc[Configure Tables]
- ** xref:dashboard-custom-config-text.adoc[Configure Text]
- ** xref:dashboard-config-ref.adoc[Dashboard Configuration Reference]
-* xref:traces-overview.adoc[Exploring Trace Data for Apps and APIs]
+* xref:dashboards.adoc[Dashboards]
```

Comment:
Rough realignment. A lot more work required on nav and underlying content to group info.